index.html

```diff
@@ -110,7 +110,7 @@ <h2>Stages</h2>
     <td>The solution is complete and no further work is possible without implementation experience, significant usage and external feedback.
     <td>Complete: all semantics, syntax and API are completed described
     <td>Limited: only those deemed critical based on implementation experience
-    <td>Spec compliant
+    <td>Spec-compliant polyfills, engines, and any other implementations (implementations that modify global built-in objects may be tentatively published)
```

Comment:
I think we should reference the w3c on this: https://www.w3.org/2001/tag/doc/polyfills/#feature-life-cycle-and-the-role-of-polyfills and if this is ambiguous, highlight that polyfills are only recommended to be shipped _after_ browsers have shipped. This covers a lot of the issues that come up.

Reply:
Thanks! I love the idea of linking here and also highlighting that recommendation, and that matches what we discussed in plenary.